_schemaV2.graphql

```diff
@@ -10547,6 +10547,12 @@ type DismissTaskSuccess {
   task: Task!
 }
 
+# Display texts for the order based on its state
+type DisplayTexts {
+  # Title text to display for the order
+  titleText: String!
+}
+
```

Comment:
If we have a solid buyer state type then I think we can keep a completely visual concern like a text string on the client. For example we might show something different in a text view based on the viewport size.

Reply:
We already have at least 2 places Force and Eigen where we need to show exact same strings. And the message part will have a bit more logic in integrating other order properties (shipping/dates) into the display. The idea here is that we don't have to repeat this code on the client and will just keep adding whatever needed here in this DisplayTexts.